src/LsmTree.hs

```diff
@@ -0,0 +1,33 @@
+module LsmTree
```

Comment:
You also need to add this module to `.cabal` file so it will be built by `cabal` or `stack`